core/templates/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -120,6 +120,19 @@ angular.module('oppia').directive('objectEditor', [
         };
         if (directiveName) {
           if (MIGRATED_EDITORS.indexOf(directiveName) >= 0) {
+            if (directiveName === (
+              'list-of-sets-of-translatable-html-content-ids'
+            ) && !scope.initArgs) {
+              throw new Error(
+                '\nProvided initArgs: ' + scope.initArgs +
+                '\nExpected initArgs: To be an object containing ' +
+                'a property called \'choices\' whose value would be an ' +
+                'array of answer choices in drag and drop response.' +
+                '\nSince this initArgs will be passed into ' +
+                'list-of-sets-of-translatable-html-content-ids editor, ' +
+                'it might be the cause of issue #14708'
```

Comment:
You can add this part / info in the issue thread instead.

Reply:
Should I also remove this part from here?

Reply:
Yes because it seems unnecessary to log so much text on the backend if we can have the record of it on the issue thread.

Reply:
done